guides/source/i18n.md

```diff
@@ -135,6 +135,8 @@ I18n.available_locales = [:en, :pt]
 I18n.default_locale = :pt
 ```
 
+Note that appending directly to `I18n.load_paths` instead of the application configured i18n will _not_ override translations from external gems.
```

Comment:
I'm Spanish and I don't speak English natively, but I would say this reads better to me:
```suggestion
Note that appending directly to `I18n.load_paths` instead of to the application's configured i18n will _not_ override translations from external gems.
```

Reply:
Sure, I'll use that. I don't want this change to be multiple commits, so I'll just give you credit in the commit message for the contributors app to pick up on instead.

Reply:
Thanks @gmcgibbon !

Sadly credit went to the wrong place, it went here -> [Alberto Almagro Sotelo](https://contributors.rubyonrails.org/contributors/alberto-almagro-sotelo/commits) but my commits are really here -> [Alberto Almagro](https://contributors.rubyonrails.org/contributors/alberto-almagro/commits).

How can we fix that?